wheels/model/nestedproperties.cfm

```diff
@@ -66,12 +67,15 @@ public boolean function $validateAssociations(required boolean callbacks) {
 			}
 			if (IsArray(local.array)) {
 				for (local.i = 1; local.i <= ArrayLen(local.array); local.i++) {
-					$invoke(componentReference = local.array[local.i], method = "valid", invokeArgs = arguments);
+					local.isAssociationValid = local.array[local.i].valid(callbacks = arguments.callbacks,validateAssociations = arguments.validateAssociations);
```

Comment:
I had to remove the `$invoke` call in favor of running `.valid` manually because it was not invoking the `model.valid` function on the nested associations for whatever reason when doing it via `$invoke`